canopy_bioemi_mod.F90

```diff
@@ -0,0 +1,310 @@
+        REAL(RK),          PARAMETER     :: BTEMP_4_SUN     =  1.050_rk   !...
+        REAL(RK),          PARAMETER     :: BTEMP_5_SUN     =  1.051_rk   !...
+        REAL(RK),          PARAMETER     :: CTEMP_1_SUN     =  1.083_rk   !Exponential 2-m PPFD --> PPFD parameters (Level 1 =
+        !top of canopy
+        REAL(RK),          PARAMETER     :: CTEMP_2_SUN     =  1.096_rk   !Based on Table 1 in Silva et al. (2022)
+        REAL(RK),          PARAMETER     :: CTEMP_3_SUN     =  1.104_rk   !
+        REAL(RK),          PARAMETER     :: CTEMP_4_SUN     =  1.098_rk   !
+        REAL(RK),          PARAMETER     :: CTEMP_5_SUN     =  1.090_rk   !...
+        REAL(RK),          PARAMETER     :: DTEMP_1_SUN     =  0.002_rk   !...
+        REAL(RK),          PARAMETER     :: DTEMP_2_SUN     =  -0.128_rk  !...
+        REAL(RK),          PARAMETER     :: DTEMP_3_SUN     =  -0.298_rk  !...
+        REAL(RK),          PARAMETER     :: DTEMP_4_SUN     =  -0.445_rk  !...
+        REAL(RK),          PARAMETER     :: DTEMP_5_SUN     =  -0.535_rk  !...
+        REAL(RK),          PARAMETER     :: ATEMP_1_SHADE   =  -12.846_rk !...
+        REAL(RK),          PARAMETER     :: ATEMP_2_SHADE   =  -11.343_rk !...
+        REAL(RK),          PARAMETER     :: ATEMP_3_SHADE   =  -1.068_rk  !...
+        REAL(RK),          PARAMETER     :: ATEMP_4_SHADE   =  -5.551_rk  !...
+        REAL(RK),          PARAMETER     :: ATEMP_5_SHADE   =  -5.955_rk  !...
+        REAL(RK),          PARAMETER     :: BTEMP_1_SHADE   =  1.060_rk   !...
+        REAL(RK),          PARAMETER     :: BTEMP_2_SHADE   =  1.053_rk   !...
+        REAL(RK),          PARAMETER     :: BTEMP_3_SHADE   =  1.031_rk   !...
+        REAL(RK),          PARAMETER     :: BTEMP_4_SHADE   =  1.051_rk   !...
+        REAL(RK),          PARAMETER     :: BTEMP_5_SHADE   =  1.053_rk   !...
+        REAL(RK),          PARAMETER     :: CTEMP_1_SHADE   =  0.871_rk   !...
+        REAL(RK),          PARAMETER     :: CTEMP_2_SHADE   =  0.890_rk   !...
+        REAL(RK),          PARAMETER     :: CTEMP_3_SHADE   =  0.916_rk   !...
+        REAL(RK),          PARAMETER     :: CTEMP_4_SHADE   =  0.941_rk   !...
+        REAL(RK),          PARAMETER     :: CTEMP_5_SHADE   =  0.956_rk   !...
+        REAL(RK),          PARAMETER     :: DTEMP_1_SHADE   =  0.015_rk   !...
+        REAL(RK),          PARAMETER     :: DTEMP_2_SHADE   =  -0.141_rk  !...
+        REAL(RK),          PARAMETER     :: DTEMP_3_SHADE   =  -0.368_rk  !...
+        REAL(RK),          PARAMETER     :: DTEMP_4_SHADE   =  -0.592_rk  !...
+        REAL(RK),          PARAMETER     :: DTEMP_5_SHADE   =  -0.743_rk  !...
+
+        REAL(RK),          PARAMETER     :: CT2             =  230.0_rk   !Deactivation energy (kJ/mol) (Guenther et al., 2012)
+
+!Calculate photolyis shading/correction factor through canopy, i.e., the fraction of sunlit leaves downward through canopy
+
+        call canopy_phot(FCLAI, LAI, CLU, COSZEN, RJCF)
```

Comment:
In the clumping case, relative direct beam irradiance (our current photolysis attenuation factor) is not equivalent to sunlit leaf fraction. See Bonan eq. 14.18.

Without clumping, it would be. But in general they aren't necessarily the same (e.g. with different radiative transfer approach).

Reply:
@zmoon Thank you for that.  We do get very reasonable sunlit/shaded fractions that look like this from Clifton et al.:

![image](https://user-images.githubusercontent.com/26631222/224439675-a9ad6fe0-36c9-4f29-a7d1-ba95c575a3c4.png)


But, if not fully correct, please go ahead and make proposed changes and commit so we can test, thanks!

Reply:
@drnimbusrain I updated in d065c28

Reply:
@zmoon I did run a quick check and the isoprene emissions are rather consistent:  
![image](https://user-images.githubusercontent.com/26631222/229370862-8c11da8b-fb9f-492e-8942-b3c2db973687.png)

I'd say this is a good change to be consistent approach with Bonan! Nice work.